ddl_script/08012021_alter_user_table.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE user DROP COLUMN firebase_uid;
```

Comment:
you forget to add `password` column

Reply:
solved in 007618a918bc240189032ba2189f1136c681a8f9